# HG changeset patch
# User Markus Mützel <markus.muetzel@gmx.de>
# Date 1711637836 -3600
#      Thu Mar 28 15:57:16 2024 +0100
# Node ID d00233197c4c4b90b31bc5adc5241deb101f8693
# Parent  7994f988b7b8900368a6575ac1b9f437cf860566
strel.cc: Include <cassert> in file that uses "assert" (bug #65525).

* src/strel.cc: The header <cassert> happened to be included via some headers
from Octave before. That is no longer the case for Octave 10. Include the
header directly in a file that requires it.

diff -r 7994f988b7b8 -r d00233197c4c src/strel.cc
--- a/src/strel.cc	Sat Jan 20 14:15:28 2024 +0000
+++ b/src/strel.cc	Thu Mar 28 15:57:16 2024 +0100
@@ -20,6 +20,7 @@
 
 #include "strel.h"
 
+#include <cassert>
 #include <vector>
 
 #include <octave/oct.h>
